searx/version.py

```diff
@@ -73,7 +74,8 @@ def get_git_version():
 
 
 try:
-    from searx.version_frozen import VERSION_STRING, VERSION_TAG, GIT_URL, GIT_BRANCH
+    vf = importlib.import_module('searx.version_frozen')
+    VERSION_STRING, VERSION_TAG, GIT_URL, GIT_BRANCH = vf.VERSION_STRING, vf.VERSION_TAG, vf.GIT_URL, vf.GIT_BRANCH
```

Comment:
Strange, why does pyright report this:

```
  searx/utils.py:26:27 - error: "VERSION_TAG" is unknown import symbol (reportGeneralTypeIssues)
  searx/version.py:77:10 - warning: Import "searx.version_frozen" could not be resolved (reportMissingImports)
  searx/webapp.py:93:27 - error: "VERSION_STRING" is unknown import symbol (reportGeneralTypeIssues)
  searx/webapp.py:93:43 - error: "GIT_URL" is unknown import symbol (reportGeneralTypeIssues)
  searx/webapp.py:93:52 - error: "GIT_BRANCH" is unknown import symbol (reportGeneralTypeIssues)
  searx/infopage/__init__.py:36:23 - error: "GIT_URL" is unknown import symbol (reportGeneralTypeIssues)
```

when import the names from searx.version_frozen ..  there are fallback values in line 10

https://github.com/searxng/searxng/blob/ededaab807a22abdd69a7c802cda3bb256d6ab51/searx/version.py#L10-L15